conans/client/tools/win.py

```diff
@@ -117,8 +117,11 @@ def latest_vs_version_installed(output):
 def msvs_toolset(settings):
     toolset = settings.get_safe("compiler.toolset")
     if not toolset:
-        vs_version = settings.get_safe("compiler.version")
-        toolset = MSVS_DEFAULT_TOOLSETS.get(vs_version)
+        compiler_version = settings.get_safe("compiler.version")
+        if settings.get_safe("compiler") == "intel":
+            toolset = "Intel C++ Compiler " + compiler_version
```

Comment:
Need the ".0" here like everywhere else :-).

Reply:
yeah, keep forgetting about it :) fixed now